pyartcd/pyartcd/pipelines/okd_images_health.py

```
@@ -509,11 +516,113 @@ def url_text(self, url, text):
+            )
+
+        prompt_parts.extend(
+            [
+                '\n*Instructions:*',
+                '1. Analyze failure logs for each image',
+                '2. Determine if fix is possible via ocp-build-data metadata changes only (Dockerfile patches, environment variables, build args, etc.)',
+                '3. **DO NOT** push fixes to upstream source repositories - only modify ocp-build-data',
+                '4. Only proceed if confidence level >= 90%',
+                '5. If proceeding:',
+                f'   - Push fix to your ocp-build-data fork for group `openshift-{version}` with `--variant=okd`',
+                f'   - **All image metadata changes MUST be under the `okd:` stanza only**',
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)